src/widgets/buttonmatrix/lv_buttonmatrix.c

```diff
@@ -456,14 +456,17 @@ static void lv_buttonmatrix_event(const lv_obj_class_t * class_p, lv_event_t * e
         }
     }
     else if(code == LV_EVENT_PRESSING) {
-        /*If a slid to a new button, discard the current button and don't press any buttons*/
```

Comment:
Please keep this comment

Reply:
The comment is preserved i justed moved it to the position where the type of input device is known.

Reply:
I'm blind, sorry :smile: